src/containers/views/DataFileEdit.js

```diff
@@ -82,47 +124,46 @@ export class DataFileEdit extends Component {
       return <h1>{getNotFoundMessage("data file")}</h1>;
     }
 
-    const { path, raw_content } = datafile;
+    const { path, raw_content, content } = datafile;
     const filename = getFilenameFromPath(path);
 
     const keyboardHandlers = {
       'save': this.handleClickSave,
     };
 
+    const editorView = this.state.guiView ? " hidden" : "";
+    const uiView = this.state.guiView ? "" : " hidden";
```

Comment:
What about rendering according to `guiView` instead of adding hidden class? For example;

```javascript
if(this.state.guiView) {
  return (
    <div id="raw-editor">...
  );
} else {
  return (
    <div id="gui">...
  )
}